python/plugins/processing/algs/gdal/ogr2ogr.py

```diff
@@ -36,6 +37,7 @@
 class ogr2ogr(GdalAlgorithm):
     INPUT = 'INPUT'
     OPTIONS = 'OPTIONS'
+    LAYER = 'LAYER'
```

Comment:
This parameter name is quite misleading. something like 'KEEP_LAYERS' seems more fiting.

Reply:
I renamed the parameter as you suggested.